### ISSUE TITLE
Problems with NTFS-formatted Truecrypt volume. 

### ISSUE THREAD

**Issue by neurodroid** [neurodroid/cryptonite#38]
_From [karl.pie...@googlemail.com](https://code.google.com/u/111132289013130251129/) on August 20, 2012 10:37:07_

Hi,

I've been using cryptonite on a Galaxy Note (now updated to ICS, and rooted). I mostly use cryptonite to get access to the truecrypt binary on Android (i.e. I use it from the command line).

I went to mount my secure volume the other day - which completed Ok, but when I cd into the mountpoint - it's got files in it, just not _my_ files!?

e.g. from a root Shell I do:

/data/data/csh.cryptonite/truecrypt -k "" -m ro /sdcard/Files/volume.tc /sdcard/Secure

This completes without error - if I then do:

cd /sdcard/Secure
ls -l

I get:

-rwxrwxrwx root root 42023 1970-01-01 01:00 ani_upload_1_kernel_panic.jpg
-rwxrwxrwx root root 39255 1970-01-01 01:00 ani_upload_2_cp_crash.jpg
(whole bunch of other similar file names)

Basically, these are not the files I'm looking for...

Without anything mounted, 'losetup -f' gives:

/dev/block/loop0

The 'phone has full disk encryption turned on, and sdcard encryption turned on.

It looks like something is picking the 'wrong' thing up somewhere, and mounting it.

From memory - I think this all worked OK before I turned on SD encryption.

I've double checked 'volume.tc' on a PC - and it definitely contains my files - not the files the 'phone shows when it's mounted.

This is with Cryptonite 0.6.17 (installed from Google Play).

Any suggestions what I can do to fix this, or troubleshoot it further?

Thanks,

-Karl

_Original issue: http://code.google.com/p/cryptonite/issues/detail?id=38_
